tex/rep-theory/semisimple.tex

```diff
@@ -116,8 +129,47 @@ \section{Schur's lemma continued}
 \begin{proof}
 	Apply Schur's lemma to the inclusion $W \injto V$.
 \end{proof}
+Recall from \Cref{sec:vector_space_linear_maps} that a linear maps from a
+$n$-dimensional vector space to a $m$-dimensional vector space can be written as
+a $n\times m$ matrix. Here the situation is similar, however the matrices are
+made for each irrep independently, and the non-isomorphic irreps, in some sense,
+``doesn't talk to each other''.
 
+%We can compare this to \label{prop:rep_direct_sum}: When our $k$-algebra is a
+%direct sum $A\oplus B$, then every representation $V$ can be broken down
+%cleanly into $V_A\oplus V_B$, and there is no ``mixing'' between the action of
+%$A$ and $B$ on $V_A$ and $V_B$.
 
+\begin{remark}
+	The representation $V^{\oplus n}$ can also be viewed as $n$
+	vectors of $V$ ``stacked horizontally'', as we did in
+	\Cref{ex:mat2_representation}:
+	\[
+		\begin{pmatrix}
+			\vdots & \vdots & & \vdots \\
+			v_1 & v_2 & \cdots & v_n \\
+			\vdots & \vdots & & \vdots
+		\end{pmatrix} \in V^{\oplus n}.
+	\]
+	That way, the action is given by
+	\[
+		\begin{pmatrix}
+			\vdots & \vdots & & \vdots \\
+			v_1 & v_2 & \cdots & v_n \\
+			\vdots & \vdots & & \vdots
+		\end{pmatrix}
+		\begin{bmatrix}
+			c_{11} & c_{21} & \cdots & c_{(m-1)1} & c_{m1} \\
+			c_{12} & c_{22} & \cdots & c_{(m-1)1} & c_{m2} \\
+			\vdots & \vdots & \ddots & \vdots & \vdots \\
+			c_{1n} & c_{2n} & \cdots & c_{(m-1)n} & c_{mn}
+		\end{bmatrix}
+		\in V^{\oplus m}.
+	\]
+	It may be clearer this way to see the type mismatch happening.
+	And this also gives a natural explanation why the intertwining operators in
+	\Cref{prob:regA_intertwine} corresponds to right matrix multiplication.
```

Comment:
(does it make sense to put the problem before the theorem?)
I just mentioned the answer of the problem here though

Reply:
i think it's fine